deploy/manifests/support-bundle-manager.yaml

```diff
@@ -16,21 +16,22 @@ spec:
     metadata:
       labels:
         app: support-bundle-manager
-        harvesterhci.io/supportbundle: sample
+        rancher/supportbundle: sample
     spec:
       containers:
       - args:
         - /usr/bin/support-bundle-kit
         - manager
-        - --standalone
         env:
-        - name: NAMESPACES
+        - name: SUPPORT_BUNDLE_TARGET_NAMESPACES
           value: harvester-system,longhorn-system
-        - name: HARVESTER_SUPPORT_BUNDLE_NAME
+        - name: SUPPORT_BUNDLE_NAME
           value: sample
-        - name: HARVESTER_SUPPORT_BUNDLE_DEBUG
+        - name: SUPPORT_BUNDLE_DEBUG
           value: "true"
-        - name: HARVESTER_SUPPORT_BUNDLE_MANAGER_POD_IP
+        - name: SUPPORT_BUNDLE_WAIT_TIMEOUT
+          value: 5m
```

Comment:
Hmm, we probably cannot address it now, but we shouldn't have a timeout for generating the support bundle. As long as there is the progress made, we should continue to wait. Support bundle can be hundreds of MB big.

Reply:
@yasker understood. fixed.